taf/oll/validation.py

```diff
@@ -0,0 +1,111 @@
+import os
+import taf.oll.repositoriesdb as repositoriesdb
+from taf.oll.exceptions import InvalidBranch
+
+def validate_branch(auth_repo, target_repos, branch_name, default_branch='master'):
```

Comment:
`default_branch` param is not used

Reply:
We expect what everything will be merged into master. So, the idea was to support different default branches. I decided not to implement that now (but forgot to remove the additional parameter), but we can create an issue to generalize this validation as possible if you agree

Reply:
I just wanted to point that out because I wasn't sure if you forgot to use that parameter somewhere, or skipped implementing it for now. You can decide what is better for now

Reply:
I'll remove the parameter for now

Reply:
Created an issue for making validation more generic